display_list/display_list_enum_unittests.cc

```diff
@@ -23,5 +24,19 @@ TEST(DisplayListEnum, ToSkTileMode) {
   ASSERT_EQ(ToSk(DlTileMode::kDecal), SkTileMode::kDecal);
 }
 
+TEST(DisplayListEnum, ToDlBlendMode) {
+  ASSERT_EQ(ToDl(SkBlendMode::kSrc), DlBlendMode::kSrc);
+  ASSERT_EQ(ToDl(SkBlendMode::kLastCoeffMode), DlBlendMode::kLastCoeffMode);
+  ASSERT_EQ(ToDl(SkBlendMode::kDst), DlBlendMode::kDst);
+  ASSERT_EQ(ToDl(SkBlendMode::kLastMode), DlBlendMode::kLastMode);
+}
```

Comment:
I would do all of them. Cutting and pasting the values and then using a good regex replace macro should create lines to test all of them with just a little bit of typing.